### DISCUSSION TITLE
Migrate Flutter Drivers tests to Flutter Integration tests

### DISCUSSION THREAD

**Discussion by nilsreichardt** [firebase/flutterfire#6829] [Feature request] [answered]
Are there any plans to migrate the Flutter Driver tests to the new Flutter Integration tests?

**Comment by Salakar** [comment, marked as answer]
Yes, we'd also welcome PRs to kickstart the process if you're willing.

**Reply by nilsreichardt** [reply]
I could possibly find some time to help out 👍 But I can't guarantee anything.

**Reply by nilsreichardt** [reply]
![image](https://user-images.githubusercontent.com/24459435/169666684-e7957913-6237-4d9a-8871-8272787a51e9.png)

Maybe this will also make the CI more stable
